README.md

```diff
@@ -1,17 +1,91 @@
-## My Project
+## AWS IoT Jobs client library
+*(part of the AWS IoT Device SDK for Embedded C)*
```

Comment:
It would also be part of the `amazon-freertos` repository. Should we mention about this library just being part of the C-SDK?

Reply:
Ah, the tangled web we weave.  I think it makes sense to see this as part of the C-SDK, and the C-SDK as included in other places.